src/mesh/RadioInterface.cpp

```diff
@@ -123,6 +123,17 @@ const RegionInfo regions[] = {
     */
     RDEF(MY_919, 919.0f, 924.0f, 100, 0, 27, true, true, false),
 
+    /*
+        Singapore
+        SG_433 Band 26a: 433.05 - 434.79 MHz at 10mW, 10% duty cycle.
+        SG_868 Band 29: 868 - 869 MHz at 500mW, no restrictions.
+        SG_923 Band 30c: 917 - 925 MHz at 100mW, 1% duty cycle.
+        https://www.imda.gov.sg/-/media/imda/files/regulation-licensing-and-consultations/ict-standards/telecommunication-standards/radio-comms/imdatssrd.pdf
+    */
+    RDEF(SG_433, 433.05f, 434.79f, 10, 0, 10, true, false, false),
```

Comment:
this is `EU_433` so is not needed (see #2696).

Reply:
Removed.